### PR TITLE
[202012] cherry-picking PR#8838 to 202012 branch

### PR DESCRIPTION
**Pull request by yxieca**
#### Why I did it
Cherry-pick #8838 to 202012 branch.

#### How to verify it
Review the diff after cherry-picking

NOTE: please use rebase and merge (don't squash).

### CODE REVIEW

**Review by lguohan — APPROVED** [review]
:shipit:

File: slave.mk [hunk 1]
```diff
@@ -944,7 +949,10 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
 	export python_swss_debs+=" $(addprefix $(IMAGE_DISTRO_DEBS_PATH)/,$(LIBSWSSCOMMON)) $(addprefix $(IMAGE_DISTRO_DEBS_PATH)/,$(PYTHON_SWSSCOMMON)) $(addprefix $(IMAGE_DISTRO_DEBS_PATH)/,$(PYTHON3_SWSSCOMMON))"
 	export sonic_utilities_py3_wheel_path="$(addprefix $(PYTHON_WHEELS_PATH)/,$(SONIC_UTILITIES_PY3))"
 	export sonic_host_services_py3_wheel_path="$(addprefix $(PYTHON_WHEELS_PATH)/,$(SONIC_HOST_SERVICES_PY3))"
-
+	export components="$(foreach component,$(notdir $^),\
```
Comment (lguohan) [inline slave.mk:952]: i do not think we need this components, it is for app extension which is not on 202012.

**Review by lguohan — CHANGES_REQUESTED** [review]
remove app extension changes.

File: slave.mk (hunk 1 above)
Comment (yxieca) [reply slave.mk]: Done

**Review by lguohan — APPROVED** [review]

**Pull request merged**